merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/models/MissionModelTest.java

```diff
@@ -61,14 +51,18 @@ public void shouldGetActivityTypeList() {
             ));
 
         // WHEN
-        final Map<String, ActivityType> typeList = unconfiguredMissionModel.getActivityTypes();
+      final var activityTypes = new HashMap<String, ActivityType>();
+      registry.taskSpecTypes().forEach((name, specType) -> {
+        activityTypes.put(name, new ActivityType(name, specType.getParameters(), specType.getRequiredParameters(), specType.getReturnValueSchema()));
+      });
+      final Map<String, ActivityType> typeList = activityTypes;
 
         // THEN
         assertThat(typeList).containsAllEntriesOf(expectedTypes);
     }
```

Comment:
This block seems to have been dedented relative to the surrounding code, probably because this file was originally set at 4-space indentation. I'm fine with either, as long as it's consistent :)

(This happened throughout this file.)